pkg/plugins/golang/v3/scaffolds/init.go

```diff
@@ -30,9 +30,9 @@ import (
 
 const (
 	// ControllerRuntimeVersion is the kubernetes-sigs/controller-runtime version to be used in the project
-	ControllerRuntimeVersion = "v0.10.0"
+	ControllerRuntimeVersion = "v0.11.0"
 	// ControllerToolsVersion is the kubernetes-sigs/controller-tools version to be used in the project
-	ControllerToolsVersion = "v0.7.0"
+	ControllerToolsVersion = "v0.8.0"
```

Comment:
v0.8.0 has v0.23 version in go mod: https://github.com/kubernetes-sigs/controller-tools/releases/tag/v0.8.0